libpod/oci_conmon_common.go

```diff
@@ -1189,7 +1183,10 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 	}
 
 	// 0, 1 and 2 are stdin, stdout and stderr
-	conmonEnv := r.configureConmonEnv(runtimeDir)
+	conmonEnv, err := r.configureConmonEnv()
+	if err != nil {
+		return 0, fmt.Errorf("configuring conmon env: %w", err)
+	}
```

Comment:
LGTM. I've been trying my best to find a way to break this, like setting `IFS` or other nasties, and can't find any problems.

One suggestion for cleanup: if you scroll down a few lines, to 1200, `cmd.Env = r.conmonEnv`, that seems to be a NOP. `r.conmonEnv` is set above to `.Engine.ConmonEnvVars`, which in turn is defined in https://github.com/containers/podman/blob/94f47d6f6641b20658d06b1c80e0191d81d40ce0/libpod/options.go#L191-L192 ... which in turn does not seem to be invoked from anywhere. It dates back to the 2017 crio import, and I think that's all dead code. A future maintainer might appreciate not having to waste time chasing that wild goose.

Reply:
Great catch!  Ough ... I really don't like how we threw containers.conf over the wall and years after still find (on a weekly basis) that many options are not tested at all :(  Rushed things turn into ghosts :ghost:

Reply:
@edsantiago, I made a note to fix this in another PR.  I think it's time to sit down and check which containers.conf options/fields are actually used and then decide case-by-case what to do with them.

Reply:
@baude something for 5.0.